### DISCUSSION TITLE
Welcome to pyspark-ai Discussions!

### DISCUSSION THREAD

**Discussion by gengliangwang** [pyspark-ai/pyspark-ai#96] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pranavchiku** [comment]
Hello, I am Pranav Goswami, a final year undergrad at the computer science department, Indian Institute of Technology, Jodhpur. I am doing open source for more than a year, currently contributing to development on Fortran compiler @lfortran. I recently completed my GSOC '23 program which was "Compiling SciPy with LFortran" under @fortran-lang. I pursued a summer internship with Warner Bros. Discovery from which I got to know about pyspark-ai, I developed a feature engineering platform there. I am highly interested in contributing to development of pyspark-ai, do let me know if there is work to do :)

**Reply by gengliangwang** [reply]
@Pranavchiku Thanks for your interest! You can start with building API docs or increasing the test coverage.

**Comment by muhammadfaraz786** [comment]
hello there ! is there any way to use ai.plot in web application where i can convert that dash plotly code to json and then render it
